client/modules/reconportal/src/LeafletMap/LeafletMap.tsx

```diff
@@ -9,26 +9,40 @@ import {
   Marker,
   Popup,
 } from 'react-leaflet';
-import { faMap } from '@fortawesome/free-solid-svg-icons';
+import { faMap, faLocationDot } from '@fortawesome/free-solid-svg-icons';
 import { OpenTopoPopup } from './OpenTopoPopup';
 import {
   createSvgMarkerIcon,
   getOpenTopoColor,
+  getReconEventColor,
   ZoomConditionalLayerGroup,
+  createClusterIcon,
+  ZoomOnEventSelection,
 } from './leafletUtil';
 import { getFirstLatLng } from './utils';
-import { useGetOpenTopo } from '@client/hooks';
+import {
+  useGetReconPortalEvents,
+  type ReconPortalEvents,
+  useGetOpenTopo,
+  useReconEventContext,
+  getReconPortalEventIdentifier,
+} from '@client/hooks';
+import MarkerClusterGroup from 'react-leaflet-markercluster';
 
 /* no need to import leaflet css as already in base index
 import 'leaflet/dist/leaflet.css';
 */
 
 import styles from './LeafletMap.module.css';
+import { ReconPortalPopup } from './ReconPortalPopUp';
 
 export const mapConfig = {
   startingCenter: [40, -80] as L.LatLngTuple,
   minZoom: 2, // 2 typically prevents zooming out too far to see multiple earths
   maxZoom: 24, // Maximum possible detail
+  maxFitBoundsInitialZoom: 18,
+  maxFitBoundsSelectedFeatureZoom: 18,
```

Comment:
```suggestion
  maxFitBoundsSelectedFeatureZoom: 18,
```

use in `maxFitBoundsSelectedFeatureZoom` in `MarkerClusterGroup` or drop?